main/test/src/main/MainModuleTests.scala

```diff
@@ -0,0 +1,50 @@
+package mill.main
+
+import mill.api.Result
+import mill.{Agg, T}
+import mill.util.{TestEvaluator, TestUtil}
+import utest.{TestSuite, Tests, test}
+
+object MainModuleTests extends TestSuite {
+
+  object mainModule extends TestUtil.BaseModule with MainModule {
+    def hello = T { Seq("hello") }
+    def hello2 = T { Seq("hello2") }
```

Comment:
minor: would make the returned value different from the target name so that we can differentiate them in the test